integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonTableCompactor.scala

```diff
@@ -373,11 +381,7 @@ class CarbonTableCompactor(carbonLoadModel: CarbonLoadModel,
       } else {
         true
       }
-      // here either of the conditions can be true, when delete segment is fired after compaction
-      // has started, statusFileUpdation will be false , but at the same time commitComplete can be
```

Comment:
here may be instead of deleting the complete comment, you can rewrite the comment required for `commitComplete`

Reply:
the existing comment in the code was that, why we need to consider both the flag. Now only one flag is there. so, it is understandable. 

Based on new code if it is not understandable and if you have any suggestion for comment, you can give here. I will add.